README.md

```diff
@@ -253,7 +253,7 @@ Sikeres lefutás után ellenőrizzük valamelyik lekérdező művelettel, hogy v
 
 1.  Lásd el a választ dokumentáló `ProducesResponseType` attribútummal a `DeleteTermek` függvényt. A kliens ugyanolyan válaszokra számíthat, mint módosítás esetén.
 
-1. Mivel a függvény eddig sem kapott vagy adott vissza entitástípust, így a megvalósításon nem kell módosítani. Ellenőrizd a Swagger felületen, hogy a specifikációnak megfelelő lehetséges válaszok (201, 404) megjelennek-e. Hajts végre egy beszúrást a Swagger felületen, majd a beszúrt elemet töröl ki, a törlő művelettel.
+1. Mivel a függvény eddig sem kapott vagy adott vissza entitástípust, így a megvalósításon nem kell módosítani. Ellenőrizd a Swagger felületen, hogy a specifikációnak megfelelő lehetséges válaszok (204, 404) megjelennek-e. Hajts végre egy beszúrást a Swagger felületen, majd a beszúrt elemet töröl ki, a törlő művelettel.
```

Comment:
Lehet ugyanúgy 200 is

Reply:
Ahány ház, annyi szokás, a REST csak egy ajánláshalmaz :) [MS ajánlás](https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design#delete-methods)